lib/ansible/module_utils/urls.py

```diff
@@ -1023,3 +1024,41 @@ def fetch_url(module, url, data=None, headers=None, method=None,
         info.update(dict(msg="An unknown error occurred: %s" % str(e), status=-1))
 
     return r, info
+
+def fetch_file(module, url, data=None, headers=None, method=None,
+              use_proxy=True, force=False, last_mod_time=None, timeout=10, bufsize=65536):
```

Comment:
I don't think that any callers currently need to modify bufsize, do they?  If so, let's make it a module constant instead of a parameter.  We can always add a new keyword argument in a backwards compatible way but we can't take it away.